.github/workflows/main.yml

```diff
@@ -22,3 +22,46 @@ jobs:
         node:
           - lts/erbium
           - node
+  canary:
+    name: canary / ${{matrix.package}} / ${{matrix.node}} on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    steps:
+      - name: checkout unified
+        uses: actions/checkout@v2
+      - name: setup node
+        uses: dcodeIO/setup-node-nvm@master
+        with:
+          node-version: ${{matrix.node}}
+      - name: setup unified
+        run: |
+          npm install -g npm
+          npm install
+          npm run build
+      - name: checkout ${{matrix.package}}
+        uses: actions/checkout@v2
+        with:
+          repository: ${{matrix.package}}
+          path: canary/${{matrix.package}}
+      - name: test ${{matrix.package}}
+        run: |
+          npm install
+          npx rimraf "node_modules/**/unified"
+          npm test
+        working-directory: canary/${{matrix.package}}
+    strategy:
+      fail-fast: false
+      max-parallel: 2
+      matrix:
+        os:
+          - ubuntu-latest
+        node:
+          - node
+        package:
+          - unifiedjs/unified-engine
+          - unifiedjs/unified-args
+          - unifiedjs/unified-language-server
+          - remarkjs/remark
+          - rehypejs/rehype
+          - retextjs/retext
+          - remarkjs/react-markdown
+          - mdx-js/mdx
```

Comment:
MDX is *really* slow to build. `unified-args` and `unified-language-server` include `unified-engine`. `retext` isn’t much different from remark and rehype. I want to guard against false positives and long builds.

What about:

```
unified-engine
remark
rehype
retext
```

^-- the key direct users of unified?

Reply:
updated in 65e7911